docs/migrating_from_genshi.rst

```diff
@@ -18,8 +18,8 @@ identical to those of Genshi_.
  * ``py:strip``
  * ``xi:include`` -- renamed ``py:include``
 
-Note that, in particular, ``py:match`` is not supported.  But Kajiki
-supports the following additional directives:
+Note that, in particular, ``py:match`` is not supported and instead implements PEP622.
```

Comment:
Note that, in particular, ``py:match`` in Kajiki differs from Genshi, implementing PEP622 (link to the PEP).

Kajiki also supports the following additional directives not in Genshi: